packages/luna/src/components/Chat/ChatMessage/ChatMessage.tsx

```diff
@@ -1,29 +1,79 @@
 import { Void } from "@litespace/types";
-import React, { useState } from "react";
+import React, { useMemo, useState } from "react";
 import cn from "classnames";
 import More from "@litespace/assets/More";
 import { Menu } from "@/components/Menu";
 import { useFormatMessage } from "@/hooks";
 import MessageEdit from "@litespace/assets/MessageEdit";
 import Trash from "@litespace/assets/Trash";
 import { Typography } from "@/components/Typography";
+import MessageCap from "@litespace/assets/MessageCap";
+import AlertCircle from "@litespace/assets/AlertCircle";
+import Send2 from "@litespace/assets/Send2";
+
+/**
+ * There are now 4 states of a message:
+ *  - Sent  => (no props)
+ *  - Pending => pending: true, error: false
+ *  - Error => pending: false, error: true
+ *  - Retrying => pending: true, error:true
+ *
+ * This is beside the 2 types of message:
+ *  - Owner: The messages you send to the user
+ *  - Reciever: The messages You get from the other user
+ */
 
```

Comment:
Don't leave an empty line between the JS Docs and the component. This way the JS Docs will not work.